Sources/GitHubAPI/GitHubAPISession.swift

```diff
@@ -18,11 +20,14 @@ public final class GitHubAPISession {
         
         self.session = URLSession(configuration: sessionConfiguration)
         self.repository = repository
+        self.prerelease = prerelease
     }
     
-    public convenience init(repository: String, apiToken: String) {
+    public convenience init(repository: String, apiToken: String, prerelease: Bool) {
         self.init(sessionConfiguration: .ephemeral,
-                  repository: repository, apiToken: apiToken)
+                  repository: repository,
+                  apiToken: apiToken,
+                  prerelease: prerelease)
     }
     
     public func latestRelease() async throws -> String {
```

Comment:
should we not pass the pre-release flag into this method directly rather than into the initialiser of the object? It's not relevant to other usages of the object (createReference, compare)

```suggestion
    public func latestRelease(prerelease: Bool = false) async throws -> String {
```

Reply:
Done